### DISCUSSION TITLE
[FEATURE REQUEST] a consistent flow to share keys with the wallet

### DISCUSSION THREAD

**Discussion by sidvishnoi** [interledger/rafiki#3126] [General]
See https://github.com/interledger/rafiki/discussions/2803 and https://github.com/interledger/web-monetization-extension/issues/311.

Filing here as it's a long term solution we're after. For the extension, we're gonna hack our way around it, so closing that issue there.

**Comment by mkurapov** [comment]
@sidvishnoi can we transfer this to the testnet repo for the a long-term solution? its more of a wallet concern than a direct Rafiki concern

**Comment by sidvishnoi** [comment]
Up to you, but I'd prefer a consistent API being part of Rafiki/OP, so wallets get it for free. Making it a wallet/testnet concern means other wallets will have to follow same approach - which is kinda what we want here - a standard.

**Comment by mkurapov** [comment]
@sidvishnoi will convert this to a discussion
